README.md

```diff
@@ -213,9 +213,21 @@ server {
 
 * Make sure iprepd-nginx is seeing the real client IP. You will usually need to use something like [ngx_http_realip_module](https://nginx.org/en/docs/http/ngx_http_realip_module.html), and confirm that it is configured correctly.
 
+## Running iprepd-nginx dev environment locally
+A dev environment is provided that can be used to test iprepd-nginx with iprepd and a dummy backend service (in this case nginx with static content).
 
-## Running locally
+```
+$ make build
+$ make run_dev_env
+```
+
+Sanity tests can then be run by:
+```
+$ make build_test
+$ make e2e_test
```

Comment:
Should this be `make smoke_test`?

Reply:
As well, it seems like we use the name `smoke test` and `e2e test` interchangeably, should we use just one?

Reply:
Yea, sorry, I was going back and forth on the naming which is why its like that. I'm gonna go with smoke for now since its really just three tests to make sure we test the actual image we deploy to some degree.

Eventually I might add more e2e and was debating how much of that to set up.